README_TR.md

```diff
@@ -0,0 +1,28 @@
+![](https://raw.github.com/HaxeFlixel/haxeflixel.com/master/src/files/images/flixel-logos/flixel-addons.png)
+
+[flixel](https://github.com/HaxeFlixel/flixel) | [uzantılar](https://github.com/HaxeFlixel/flixel-addons) | [ui](https://github.com/HaxeFlixel/flixel-ui) | [demolar](https://github.com/HaxeFlixel/flixel-demos) | [araçlar](https://github.com/HaxeFlixel/flixel-tools) | [şablonlar](https://github.com/HaxeFlixel/flixel-templates) | [dökümanlar](https://github.com/HaxeFlixel/flixel-docs) | [haxeflixel.com](https://github.com/HaxeFlixel/haxeflixel.com) | [orijinal](/README.md)
```

Comment:
```suggestion
[flixel](https://github.com/HaxeFlixel/flixel) | [uzantılar](https://github.com/HaxeFlixel/flixel-addons) | [ui](https://github.com/HaxeFlixel/flixel-ui) | [demolar](https://github.com/HaxeFlixel/flixel-demos) | [araçlar](https://github.com/HaxeFlixel/flixel-tools) | [şablonlar](https://github.com/HaxeFlixel/flixel-templates) | [dökümanlar](https://github.com/HaxeFlixel/flixel-docs) | [haxeflixel.com](https://github.com/HaxeFlixel/haxeflixel.com) | [english](/README.md)
```
"orijinal" to "english" (btw "original" is the correct spelling)

Reply:
I did that in TR file because the file itself is Turkish and I thinked "orijinal" might be understood by the Turkish audience that visits the file. I know "original" is correct spelling. 
But "orijinal" is more suitable for Turkish file.

Reply:
I did that in TR file because the file itself is Turkish and I thinked "orijinal" might be understood by the Turkish audience that visits the file. I know "original" is correct spelling. 
But "orijinal" is more suitable for Turkish file.